build.sh

```diff
@@ -395,15 +401,19 @@ case "$COMMAND" in
         ;;
 
     "start-object-server")
-        # kill any object servers that are still running
-        (pgrep -f realm-object-server || true) | while read pid; do
-            kill $pid
-        done
+        kill_object_server
         ./sync/start-object-server.command
         exit 0
         ;;
 
+    "reset-object-server-between-tests")
```

Comment:
Ok, that's fine given that this shouldn't be contributing to the long test run times.